### DISCUSSION TITLE
[reown cloud] Subdomain verification is not possible

### DISCUSSION THREAD

**Discussion by pad01g** [reown-com/appkit#3459] [Q&A]
I can only verify company.com even when I wanted to register stage.company.com domain. Is it a feature or a bug?

Also, it's strange that I have to verify domain for my web wallet project.
I can understand that dapps projects need to verify domain (to prevent scams), but I'm not sure why I need to verify domain for wallet. Our wallet product is expected to be used in many websites context (like web3auth), and we don't expect every website owners register their domains in reown cloud project page.

**Comment by natalia-bel** [comment]
I have the  same issue!
